.pre-commit-config.yaml

```diff
@@ -88,7 +88,7 @@ repos:
     hooks:
       - id: ruff
         args: [--fix, --exit-non-zero-on-fix, --show-fixes]
-        files: "^notebooks/(api|tutorials|admin)"
+        exclude: packages/syft/tests/mongomock
```

Comment:
@yashgorana help me check pls. Thanks!

Reply:
mmm weird - had just made changes to clean up the regex in 4d1c349 but this line was introduced in 2b31864. unsure why but good that this was caught!